project_status_report/models/project_status.py

```diff
@@ -0,0 +1,172 @@
+                                 string="Project",
+                                 required=True,
+                                 readonly=True,
+                                 states={'draft': [('readonly', False)]})
+    name = fields.Char(compute='_compute_name',
+                       store=True,
+                       readonly=True,
+                       states={'draft': [('readonly', False)]})
+    cost_status = fields.Selection(selection=INDICATOR_STATUS,
+                                   readonly=True,
+                                   states={'draft': [('readonly', False)]})
+    cost_color = fields.Char(readonly=True,
+                             default='#00ff00',
+                             states={'draft': [('readonly', False)]})
+    cost_remarks = fields.Html(readonly=True,
+                               states={'draft': [('readonly', False)]})
+    quality_status = fields.Selection(selection=INDICATOR_STATUS,
+                                      readonly=True,
+                                      states={'draft': [('readonly', False)]})
+    quality_color = fields.Char(readonly=True,
+                                default='#00ff00',
+                                states={'draft': [('readonly', False)]})
+    quality_remarks = fields.Html(readonly=True,
+                                  states={'draft': [('readonly', False)]})
+    delay_status = fields.Selection(selection=INDICATOR_STATUS,
+                                    readonly=True,
+                                    states={'draft': [('readonly', False)]})
+    delay_color = fields.Char(readonly=True,
+                              default='#00ff00',
+                              states={'draft': [('readonly', False)]})
+    delay_remarks = fields.Html(readonly=True,
+                                states={'draft': [('readonly', False)]})
+    global_status = fields.Selection(selection=INDICATOR_STATUS,
+                                     readonly=True,
+                                     states={'draft': [('readonly', False)]})
+    global_color = fields.Char(readonly=True,
+                               default='#00ff00',
+                               states={'draft': [('readonly', False)]})
+    global_remarks = fields.Html(readonly=True,
+                                 states={'draft': [('readonly', False)]})
+    risks = fields.Html(readonly=True,
+                        states={'draft': [('readonly', False)]})
+    user_id = fields.Many2one(comodel_name='res.users',
+                              related='project_id.user_id',
+                              store=True,
+                              readonly=True)
+    indicator_ids = fields.One2many('project.status.indicator.value',
+                                    'report_id',
+                                    readonly=True,
+                                    states={'draft': [('readonly', False)]})
+    task_snapshot_ids = fields.One2many(
```

Comment:
usually I prefer exactly this kind of indentation for fields. Would you mind adjusting the fields above like this one?
Or if you prefer move this one to the style above to make them homogeneous... but it will be a :-1: :stuck_out_tongue: